packages/mui-material/src/Link/Link.js

```diff
@@ -157,13 +158,13 @@ const Link = React.forwardRef(function Link(inProps, ref) {
     <LinkRoot
       className={clsx(classes.root, className)}
       classes={TypographyClasses}
-      color={color}
       component={component}
       onBlur={handleBlur}
       onFocus={handleFocus}
       ref={handlerRef}
       ownerState={ownerState}
       variant={variant}
+      sx={{ color: colorTransformations[color] || color, ...sx }}
```

Comment:
👍 this makes total sense, we should intercept the `sx` prop probably on all components that inherit from another.